### PR TITLE
Update Security_Cassandra_TLS.md

### PR DESCRIPTION
**Pull request by ThibaultHeylen**
added important note at top, that can cause issues

### CODE REVIEW

**Comment by ThibaultHeylen** [comment]
@BrentSLC If you can validate if the note at the top of the page is enough? And if the text is clear enough

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Security_Cassandra_TLS.md [hunk 1]
```diff
@@ -77,8 +81,12 @@ To generate the certificates, you will need two tools: *openssl* and the *Java k
 1. Now that you have certificates for every node, digitally sign them with the private key of the root CA certificate. To do so, first create a certificate signing request (CSR).
 
    ```txt
-   keytool -certreq -keystore <NODE IP>.jks -alias <NODE IP> -file <NODE IP>.csr -keypass <STRONG PASSWORD> -storepass <STRONG PASSWORD> -ext SAN=DNS:<NODE NAME>,IP:<NODE IP>
+   keytool -certreq -keystore <HOSTNAME>.jks -alias <HOSTNAME> -file <HOSTNAME>.csr -keypass <STRONG PASSWORD> -storepass <STRONG PASSWORD> -ext SAN=DNS:<NODE NAME>,IP:<NODE IP>
    ```
+   
+> [!NOTE]
+> It's advised to use hostname instead of the IP. If you choose the IP over the hostname, you have to make sure the IP can be resolved, or the SAN has the correct hostname.
```
Comment (BrentSLC) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Security_Cassandra_TLS.md:88]: Even with SAN, there should be some DNS resolution linking the IP to the CN of the cert (cassandra always does a hostname lookup)

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Security_Cassandra_TLS.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ uid: Security_Cassandra_TLS
 > [!TIP]
 > If you do not want the hassle of maintaining the DataMiner storage databases yourself, we recommend using [DataMiner Storage as a Service](xref:STaaS) instead.
 
+> [!NOTE]
+> We strongly advise to use certificates based on hostname and not IP. These hostnames should then be used in the configuration of db.xml that you can find in Connecting with DataMiner below. This recommendation is for self signed  certificates and purchased certifcates.
+> In case you would want to use the IP in the certificate and db.xml, you have to make sure that you can resolve the IP to a hostname on the network, or that the SAN is correctly set in the certificate.
```
Comment (BrentSLC) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Security_Cassandra_TLS.md:12]: Even with SAN, there should be some DNS resolution linking the IP to the CN of the cert (cassandra always does a hostname lookup)

**Review by BrentSLC — COMMENTED** [review]
Not entirely correct

**Review by BrentSLC — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**